lib/containers/common.pm

```diff
@@ -262,4 +262,35 @@ sub test_rpm_db_backend {
     }
 }
 
+sub check_containers_connectivity {
+    my $runtime = shift;
+    record_info "connectivity", "Checking that containers can connect to the host, to each other and outside of the host";
+    my $container_name = 'sut_container';
+
+    # Run container in the background (sleep for 30d because infinite is not supported by sleep in busybox)
+    assert_script_run "$runtime pull " . registry_url('alpine');
+    assert_script_run "$runtime run -id --rm --name $container_name -p 1234:1234 " . registry_url('alpine') . " sleep 30d";
+    my $container_ip = container_ip $container_name, $runtime;
+
+    # Connectivity to host check
+    my $container_route = container_route($container_name, $runtime);
+    assert_script_run "ping -c3 " . $container_route;
```

Comment:
Please do a `script_retry` on all ping commands with at `retry => 3, delay => 120`, otherwise we have to do this in a month or so anyways.

I would do with for ALL `ping` commands, external and internal. Experience has shown us that not doing so always leads to pain.

Reply:
or just `ping -c1000`  :)

Reply:
That would prolong the test for hours.

Reply:
> That would prolong the test for hours 😄

yep, sorry, it was a joke :p